serving-api/src/main/scala/com/stratio/sparkta/serving/api/actor/ControllerActor.scala

```diff
@@ -61,15 +61,23 @@ class ControllerActor(streamingContextService: StreamingContextService,
   ))
 
   val jobServerActor =
-    context.actorOf(Props(new JobServerActor(configJobServer.getString("host"),
-      configJobServer.getInt("port"))), "jobServerActor")
+    if(!configJobServer.isEmpty && !configJobServer.getString("host").isEmpty && configJobServer.getInt("port") > 0) {
+    Some(context.actorOf(Props(new JobServerActor(configJobServer.getString("host"),
+      configJobServer.getInt("port"))), "jobServerActor"))
+    } else None
+
+  val jobServerService = jobServerActor match {
```

Comment:
This is not a service, is a route. Please rename it to something like jobServerRoutes